smdebug/core/state_store.py

```diff
@@ -40,12 +41,15 @@ def __init__(self):
         if self._checkpoint_dir is not None:
             self._states_file = os.path.join(self._checkpoint_dir, METADATA_FILENAME)
             self._read_states_file()
-            self._last_seen_checkpoint_files = self._get_checkpoint_files_in_dir(
-                self._checkpoint_dir
-            )
-            self._last_seen_cp_files_size = [
-                os.path.getsize(file) for file in self._last_seen_checkpoint_files
-            ]
+            try:
+                self._last_seen_checkpoint_files = self._get_checkpoint_files_in_dir(
+                    self._checkpoint_dir
+                )
+                self._last_seen_cp_files_size = [
+                    os.path.getsize(file) for file in self._last_seen_checkpoint_files
+                ]
+            except FileNotFoundError as e:
+                logger.warning(e)
```

Comment:
self._last_seen_checkpoint_files = [] 
self._last_seen_cp_files_size = []

Reply:
why do we want to empty these two lists here?